cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,35 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
-  });
-
-  it('', () => {
+describe('Conduit', () => {
+  const userName = 'nevermore';
+  const email = 'nevermore@gmail.com';
+  const password = 'nevermore123';
+  const title = 'title';
+  const description = 'description';
+  const body = 'body';
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.login(email, userName, password);
   });
 
-  it('', () => {
+  it('should create article with valid data', () => {
+    cy.reload();
+    cy.contains('a', 'New Article').click();
+    cy.get('[placeholder="Article Title"]').type(title);
+    cy.get('[placeholder="What\'s this article about?"]').type(description);
+    cy.get('[placeholder="Write your article (in markdown)"]').type(body);
+    cy.get('.btn-primary').click();
+    cy.get('.container').should('contain', title)
+      .and('contain', 'Delete Article')
+      .and('contain', 'Edit Article');
+  });
 
+  it('should delete article', () => {
+    cy.createArticle(title, description, body);
+    cy.contains('a', 'Global Feed').click();
+    cy.contains('title').click();
+    cy.get('.article-actions').find('.btn-outline-danger').click();
+    cy.url().should('include', '/');
   });
```

Comment:
In the 'should delete article' test, you are assuming that the article you want to delete exists. However, it would be safer to create a new article at the start of this test to ensure that the article you want to delete is present. This way, your test does not depend on the state left by other tests.